Malla CIV

AbelQ97 opened the pull request.

AbelQ97 committed:
Malla CIV

etra0 commented:
Estimado, en el archivo detecté principalmente dos problemas:

* _Dibujo En Ingeniería Civil_ (s3) tiene como prerrequisito **IWG-131**, y no existe tal ramo (el más cercano es **IWI-131**)
* _Ingeniería Sísmica_ (s11) tiene como prerrequisito **CIV-337**, tampoco existe (¿quizás era **CIV-377**?)

AbelQ97 committed:
Update data_CIV.json

-Efectivamente en "Dibujo" era IWI y no IWG
-En s11: El prerequisito de Ingeniería Sísmica era CIV-331, no CIV-377 (en siga aparece así pero el ramo ese ya no existe)

AbelQ97 commented:
Corregido. Saludos.

etra0 commented:
Ahora funciona bien, lamentablemente el merge no se puede hacer por acá, debido a que en el commit eliminaste el de INF, de todas formas, se te darán los créditos respectivos
